src/test/java/com/expediagroup/hiveberg/TestIcebergInputFormat.java

```diff
@@ -82,10 +82,7 @@ public void testInputFormat() {
     shell.execute("CREATE DATABASE source_db");
     shell.execute(new StringBuilder()
         .append("CREATE TABLE source_db.table_a ")
-        .append("ROW FORMAT SERDE 'com.expediagroup.hiveberg.IcebergSerDe' ")
```

Comment:
Would it be worth having a test which uses the INPUTFORMAT directly (i.e. the current test) and then another one which tests the storage handler? So we make sure that both use cases work?

Reply:
Yeah, good shout